boards/xtensa/intel_adsp_baytrail/Kconfig.defconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_INTEL_ADSP_BAYTRAIL
+
+config BOARD
+	default "intel_adsp_baytrail"
+
+config CAVS_ICTL_0_OFFSET
+	default 6
+config CAVS_ICTL_1_OFFSET
+	default 10
+config CAVS_ICTL_2_OFFSET
+	default 13
+config CAVS_ICTL_3_OFFSET
+	default 16
+
+config MAX_IRQ_PER_AGGREGATOR
+	default 32
+
+config HEAP_MEM_POOL_SIZE
```

Comment:
this is normally a software tunable, why set it here? do we need `k_malloc()` as part of the BSP?